### PR TITLE
Write integration tests

### LINKED ISSUE #52: Write tests
**Is your feature request related to a problem? Please describe.**
Code is not tested. It's wrong.

**Describe the solution you'd like**
Use a testing library to fix this, such as Jest or TAP.

### PR DESCRIPTION
**Pull request by tanohzana**
# Fixes #52 

**Summary:**

CLI was not tested previously.

These tests bring a lot of safety and avoid breaking changes on the whole project.

**Changes:**

* Change babel presets to match the env we are using dynamically
* Add a jest config
* Add babel polyfill in all `.js` files to allow transpilation in tests 
* Change babel version (upgrade to babel 7)
* Create fixtures for tests
* Implement integration tests file

**Dependencies:**

No dependency

**Instructions:**

1. pull the branch
2. `yarn install`
3. `yarn test`

### CODE REVIEW

File: test/reacli.test.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+					const { same } = dircompare.compareSync(expectedPath, hookPath);
+					
+					expect(same).toBeTruthy()
+					expect(code).toEqual(0)
+					done()
+				})
+			})
+		})
+		
+		test("reacli hook ./scss-hook", (testDone) => {
```
Comment (gdarchen) [inline test/reacli.test.js]: Shouldn't it be `"reacli hook ./scss-hook --scss"` instead? 😊

**Review by gdarchen — CHANGES_REQUESTED** [review]
Code seems really clean to me. I guess we could maybe split the test file, what do you think about it?

And don't you think we have to check when several options are used concurrently? Such as `--redux --flow`?

Finally, would you mind writing a simple test to check that the CLI is functional when using a `.reacli` configuration file? It would then be a perfect PR 😊

**Comment by gdarchen** [comment]
- [x] feature is working
 - [ ] no breaking change was caused
 - [x] code is clean
 - [x] new tests pass

**Comment by gdarchen** [comment]
SonarCloud code coverage remains at 0%. Do you think it's normal according to the current configuration?

**Comment by tanohzana** [comment]
I thought about splitting the code, but we are going to have many other files with tests, and I made it only one file because I wanted to gather `reacli cli integration` tests together... But I'm open to making changes if you think it's better 🙂

**Comment by gdarchen** [comment]
Maybe we could split those tests in separate files but place them in a common folder so that we know where we have to write test. What do you think about it?

File: test/reacli.test.js (hunk 1 above)
Comment (tanohzana) [reply test/reacli.test.js]: ac5fef5

**Comment by tanohzana** [comment]
I do not know about Sonar.. Let's open another issue then I guess... 🤔

**Review by gdarchen — APPROVED** [review]

**Pull request merged**